File: apprefurbished/src/main/java/com/androidcollider/easyfin/managers/chart/data/ChartDataModule.java
package com.androidcollider.easyfin.managers.chart.data;

import com.androidcollider.easyfin.managers.format.number.NumberFormatManager;
import com.androidcollider.easyfin.managers.resources.ResourcesManager;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ChartDataModule {

    @Provides
    ChartDataManager provideChartDataManager(NumberFormatManager numberFormatManager, ResourcesManager resourcesManager) {
        return new ChartDataManager(numberFormatManager, resourcesManager);
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/chart/data/ChartDataModule.kt
package com.androidcollider.easyfin.common.managers.chart.data;

import android.content.Context;

import com.androidcollider.easyfin.common.managers.format.number.NumberFormatManager;
import com.androidcollider.easyfin.common.managers.resources.ResourcesManager;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ChartDataModule {

    @Provides
    ChartDataManager provideChartDataManager(NumberFormatManager numberFormatManager, ResourcesManager resourcesManager, Context context) {
        return new ChartDataManager(numberFormatManager, resourcesManager, context);
    }
}